app/controllers/orders_controller.rb

```diff
@@ -2,8 +2,8 @@ class OrdersController < ApplicationController
     
     def index
          @carts=Cart.of_user(current_user) 
-         @orders= Order.of_user(current_user)
-         @orders=@orders.sort_by(&:updated_at).reverse
+         @orders_user= Order.of_user(current_user)
+         @orders=@orders_user.sort_by(&:updated_at).reverse
```

Comment:
what exactly are you trying to do here?

Reply:
I am using them in index.html.erb file (the variables)